### ISSUE TITLE
project domain name

### ISSUE THREAD

**Issue by fricklerhandwerk** [NixOS/nix-book#23]
- use [nixpkgs.org](http://nixpkgs.org/) or [nixos.org](http://nixos.org/)?
    - related: https://github.com/NixOS/nixos-homepage/issues/633
- register nix.org?
  - [currently being squatted on](https://github.com/NixOS/nixos-homepage/issues/633#issuecomment-841634501)
- [more domain names](https://github.com/NixOS/nix-book/issues/12#issuecomment-1171484982)

@garbas this is a responsibility of the marking team, but intimately concerns documentation w.r.t. consistency and avoiding confusion

**Comment by asymmetric** [comment]
On buying nix.org, I think a good (and way cheaper) alternative could be buying nix-lang.org. It's what [many](https://www.rust-lang.org/) [other](https://elixir-lang.org/) languages do.

**Comment by fricklerhandwerk** [comment]
The Nix ecosystem and the Nix package manager are not really about the Nix language (just as Bazel is not really about Starlark). nix-lang.org would easily make the impression the project is somehow revolving around the configuration language. That would defeat the purpose of reducing confusion.

**Comment by asymmetric** [comment]
I disagree. Part of the confusion comes from using the same name for the language and the ecosystem, so renaming the language to something like "nix lang" could reduce confusion. Then, having the name is simply a good idea, whatever we decide to do with it. And potentially, it could be a place that hosts the nix-lang manual (or redirect to it)

**Comment by fricklerhandwerk** [comment]
> Part of the confusion comes from using the same name for the language and the ecosystem

This is true and we should address it with #13.

> controlling the domain is simply a good idea, whatever we decide to do with it.

Sure, it just does not help with the issue here.

**Comment by davidak** [comment]
i think it would be best to have [nix.org](https://nix.org/) for the package manager + resources about nixpkgs and nix lang and  [nixos.org](https://nixos.org/) for the linux distribution + deployment tools for it, like nixops

that would solve confusion like reported in https://github.com/NixOS/nixos-homepage/issues/832
